package-lock.json

```diff
@@ -0,0 +1,4045 @@
+{
```

Comment:
Do'nt provide a package-lock.json. We switched to yarn. (I told you on discord already :wink:)